docker-hub/api/latest.yaml

```
@@ -0,0 +1,523 @@
+swagger: '2.0'
+schemes:
+  - https
+host: hub.docker.com
+basePath: "/"
+produces:
+  - application/json
+consumes:
+  - application/json
+info:
+  title: Docker HUB API
+  version: 'beta'
+  x-logo:
+    url: https://docs.docker.com/images/logo-docker-main.png
+  description: |
+    Docker Hub is a service provided by Docker for finding and sharing container images with your team.
+    It is the world's largest library and community for container images.
+    In addition to the [Docker Hub UI](https://docs.docker.com/docker-hub/) and [Docker Hub CLI tool](https://github.com/docker/hub-tool#readme) (currently experimental),
+    Docker provides an API that allows you to interact with Docker Hub.
+    Browse through the Docker Hub API documentation to explore the supported endpoints.
+tags:
+  # Primary objects
+  - name: Authentication
+    x-displayName: Authentication
+    description: |
+      Most Docker Hub API endpoints require you to authenticate using your Docker credentials.
+      
+      Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending on your subscription (Free, Pro, or Team) and the permissions associated with it.
```

Nitpicking - I don't think it is a blocker
I am unsure about the wording (see Oleg's comments on[Mark's PR 2511](https://github.com/docker/mercury-ui/pull/2511#discussion_r597174126))
Suggestion to change into:
```suggestion
      Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending on your plan (Free, Pro, or Team) and your account's permissions.
```

---

Discussed with @usha-mandya 
Not blocking merging/release
Can be changed once language agreed upon (see Voice and Tone project)

---

OK so we're happy with `Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending on your plan (Free, Pro, or Team) and your account's permissions.` correct?

---

We're happy with either wording.
This will be reviewed and updated if needed later. Doesn't block publishing the doc.